storage/Filesystems/BtrfsSubvolumeImpl.cc

```diff
@@ -193,14 +193,14 @@ namespace storage
     MountOpts
     BtrfsSubvolume::Impl::get_default_mount_options() const
     {
-	return MountOpts(vector<string>({ "subvol=/" + path }));
+	return MountOpts(vector<string>({ "subvol=" + EtcFstab::fstab_encode("/" + path) }));
```

Comment:
The mount options are also used directly for the mount command and there the fstab encoding is wrong.